it/uk/gov/hmrc/excisemovementcontrolsystemapi/DraftExciseMovementControllerItSpec.scala

```diff
@@ -124,6 +124,36 @@ class DraftExciseMovementControllerItSpec extends PlaySpec
       postRequest(IE815).status mustBe BAD_REQUEST
     }
 
+    "remove control document references in any paths for a BAD_REQUEST" in {
+      val eisErrorResponse = "{" +
```

Comment:
do you think this could go outside the test, maybe as private val at the top of the class or as function, so we make the test shorter and less busy?

Reply:
done